wallettool/src/test/java/org/bitcoinj/wallettool/WalletToolTest.java

```diff
@@ -0,0 +1,49 @@
+package org.bitcoinj.wallettool;
+
+import org.junit.Ignore;
+import org.junit.Test;
+import picocli.CommandLine;
+
+import static org.junit.Assert.*;
```

Comment:
Maybe expand the wildcard?

Reply:
Yes, of course.

Reply:
Fixed as part of the conversion to JUnit 5: 927d7390bdcb2e01816146b8ba1646c11765aaef